device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -159,6 +171,33 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin {
             }
             
         }
+        var recurrenceRule: RecurrenceRule?
+        if(ekEvent.hasRecurrenceRules) {
+            let ekRecurrenceRule = ekEvent.recurrenceRules![0]
+            var frequency: Int64
+            switch ekRecurrenceRule.frequency {
+            case EKRecurrenceFrequency.daily:
+                frequency = 0
+            case EKRecurrenceFrequency.weekly:
+                frequency = 1
+            case EKRecurrenceFrequency.monthly:
+                frequency = 2
+            default:
+                frequency = 0
```

Comment:
Could we combine `daily` and `default` cases (as in C#)?

Reply:
can't be done like in C#